docs/gallery/general/extensions.py

```diff
@@ -326,7 +326,7 @@ class PotatoSack(MultiContainerInterface):
 from dateutil.tz import tzlocal
 
 # You can add potatoes to a potato sack in different ways
-potato_sack = PotatoSack(potatos=Potato(name='potato1', age=2.3, weight=3.0))
+potato_sack = PotatoSack(potatoes=Potato(name='potato1', age=2.3, weight=3.0))
```

Comment:
just now mentioned this one -- should I also fix `potatos` above or leave any `potatos` untouched?

Reply:
Please leave `potatos` untouched. The variable name is autogenerated, which doesn't apply 100% of the time...

Reply:
done, left potatos behind.